src/pages/about.astro

```diff
@@ -8,13 +8,15 @@ import Nav from '../components/Nav.astro';
 	<head>
 		<MainHead title="About | Fatima Guzman" description="About Fatima Guzman" />
 		<style>
-			.heroImg {
+			/* .heroImg {
```

Comment:
@FatimaGC Unless you're planning on definitely using the code, it's typically better to just delete it instead of commenting it out. Otherwise it might just end up sitting there for a while and other devs might be think the code is important in some way and not touch it themselves. 

If you ever need to look at old code for reference, you can go "back in time" by "checking out" an older commit which will contain an older version of the codebase.

For example, try running `git log` in the Terminal. You'll see something like this:

```
$ git log
commit 3646ef606592bedbee31b098fc273 (HEAD -> feature/create-account-endpoint, origin/feature/create-account-endpoint)
Author: kee-oth
Date:   Tue Jan 3 16:26:02

    Make Create Account endpoint
    
    * Regenerate Entity files

commit 0832af7abe2a4b41667c773d6f6 (origin/development, development)
Merge: e39831b 32e0
Author: kee-oth 
Date:   Tue Jan 3 09:21:24

    Merge pull request #15 from /feature/switch-api-to-use-modrs-files
    
    Feature/switch api to use modrs files

commit 32e08f64dee812360059 (origin/feature/switch-api-to-use-modrs-files)
Author: kee-oth
Date:   Tue Jan 3 09:20:02

    Delete old handlers.rs module file
```

Then, to go "back in time" to that earliest commit, I can check it out via its hash like this:
```
git checkout 32e08f64dee812360059
```

Then I can go look at or grab any needed code 👍